examples/avmnist_multimodal/config.py

```diff
@@ -0,0 +1,68 @@
+"""
+Default configurations for avmnist dataset using Multimodal Deep Learning (MMDL).
+"""
+
+from yacs.config import CfgNode as CN
+
+# -----------------------------------------------------------------------------
+# Config definition
+# -----------------------------------------------------------------------------
+
+_C = CN()
+
+# -----------------------------------------------------------------------------
+# Dataset
+# -----------------------------------------------------------------------------
+_C.DATASET = CN()
+_C.DATASET.ROOT = "avmnist"
+_C.DATASET.NAME = "data.zip"
+_C.DATASET.GDRIVE_ID = "1N5k-LvLwLbPBgn3GdVg6fXMBIR6pYrKb"
+_C.DATASET.BASE_DIR = "avmnist"
+_C.DATASET.FILE_FORMAT = "zip"
```

Comment:
Is this necessary when the extension is available?

Reply:
Yes the FILE_FORMAT is necessary as we pass this parameter to download_file_gdrive() function in the main file. No need for _C.DATASET.BASE_DIR so removed it.